### PR TITLE
Submit the first draft for FIP discussion 86 - Revert FIP-0009

### PR DESCRIPTION
**Pull request by jennijuju**
related #86

### CODE REVIEW

File: FIPS/fip-revert-fip0009.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+## Abstract
+
+Do not "refund" any burned gas from any `SubmitWindowedPoSt` message sent on-chain. 
+
+## Change Motivation
+
+We introduced the FIP-0009, which exempts `SubmitWindowedPoSt` messages that execute successfully from base-fee burn (i.e., don't burn `baseFee*gasUsed`) as a short-term stopgap to reduce the impact of rising base fee on the gas cost for continously proving existing storage in the network back in December.
+
+Since then, three long-term solutions for addressing the issue from root cause have been accepted in the network:
+- [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md), off-chain Window PoSt verification thats optimisitically accept Window PoSt proofs on-chain without verification while allowing them to be disputed later by off-chian verifiers was introduced in network v10 by actor v3. As a result, the `GasUsed` for an arbitray `SubmitWindowedPost` message has a reduction up to 90%.
```
Comment (arajasek) [inline FIPS/fip-revert-fip0009.md:26]:
```suggestion
- [FIP-0010](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0010.md), off-chain Window PoSt verification thats optimisitically accept Window PoSt proofs on-chain without verification while allowing them to be disputed later by off-chain verifiers was introduced in network v10 by actor v3. As a result, the `GasUsed` for an arbitrary `SubmitWindowedPost` message has a reduction up to 90%.
```

File: FIPS/fip-revert-fip0009.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+- [FIP-0013](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md), add `ProveCommitAggregated` method to aggragate and condensing multiple proof verifications(SNARKs) into a single aggregated message that takes up less chain bandwidth using SnarkPack has been accepted and will be introduced in the network in the upcoming [HyperDrive upgrade](https://github.com/filecoin-project/community/discussions/74#discussioncomment-707228) via actor v5. This feature will relieve the network congestion as SnarkPack can aggregate 8192 SNARK proofs, whcih allows miners to submit pro-commitments for up to 819 sectors in one `ProveCommitAggregated` message.
```
Comment (arajasek) [inline FIPS/fip-revert-fip0009.md:27]:
```suggestion
- [FIP-0013](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0013.md), add `ProveCommitAggregated` method to aggregate and condensing multiple proof verifications(SNARKs) into a single aggregated message that takes up less chain bandwidth using SnarkPack has been accepted and will be introduced in the network in the upcoming [HyperDrive upgrade](https://github.com/filecoin-project/community/discussions/74#discussioncomment-707228) via actor v5. This feature will relieve the network congestion as SnarkPack can aggregate 8192 SNARK proofs, which allows miners to submit prove-commitments for up to 819 sectors in one `ProveCommitAggregated` message.
```

File: FIPS/fip-revert-fip0009.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+- [FIP-0008](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0008.md),  adds a method for a miner to submit batched pre-commitments for multiple sectors whcih also help with increasing the chain bandwidth and relive the network congestion. This FIP is also landing in the HyperDrive Network upgrade via actor v5,
```
Comment (arajasek) [inline FIPS/fip-revert-fip0009.md:28]:
```suggestion
- [FIP-0008](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0008.md),  adds a method for a miner to submit batched pre-commitments for multiple sectors which also help with increasing the chain bandwidth and relieve the network congestion. This FIP is also landing in the HyperDrive Network upgrade via actor v5.
```

File: FIPS/fip-revert-fip0009.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+In addition, the "free" Window Posts distort the gas market and removes any incentives for miners to reduce consumption of chian bandwidth by batching as many partitions as possible into one `SubmitWindowedPoSt` message, which wastes chain bandwidth.
```
Comment (arajasek) [inline FIPS/fip-revert-fip0009.md:30]:
```suggestion
In addition, the "free" Window Posts distort the gas market and removes any incentives for miners to reduce consumption of chain bandwidth by batching as many partitions as possible into one `SubmitWindowedPoSt` message, which wastes chain bandwidth.
```

**Review by arajasek — APPROVED** [review]
LGTM with some nits

**Comment by jennijuju** [comment]
> LGTM with some nits

Thanks! Sorry I can’t type 🙏🏻

**Pull request merged**